seatunnel-e2e/pom.xml

```diff
@@ -40,6 +40,17 @@
         <junit4.version>4.13.2</junit4.version>
         <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
     </properties>
+    <dependencyManagement>
```

Comment:
Why e2e root pom need add `connector-jdbc`'s dependency?

Reply:
The driver jar is used by connector and e2e,so i want a unified version

Reply:
Just import it in `seatunnel-flink-connector-v2-e2e/connector-jdbc-flink-e2e` and `seatunnel-spark-connector-v2-e2e/connector-jdbc-spark-e2e`, not here

Reply:
Done.

Reply:
You need fixed ci

Reply:
> 

mysql and postgres e2e fail.and i have fixed it.